Update active-directory-b2c-troubleshoot-custom.md

marcelodiiorio opened the pull request:
Instead of referring to the policyName, we must refer to the "UserJourney Id" we need to invoke.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

marcelodiiorio committed:
Update active-directory-b2c-troubleshoot-custom.md

Instead of referring to the policyName, we must refer to the "UserJourney Id" we need to invoke.

MicrosoftGuyJFlo reviewed (approved):
Approved by author
